pkg/configuration/config_manager/config_handler.go

```diff
@@ -309,6 +351,13 @@ func CreateExecHandler(command []string, mountPoint string, configMeta *ConfigSp
 		return nil, err
 	}
 
+	isBatchReload := false
```

Comment:
Better to refactor into a function: `isBatchReload := isBatchMode(configMeta) `

Reply:
Fixed in bbf86cdffc028e83746c42b7fa33d25479c37ec0